content/en/docs/concepts/services-networking/service.md

```diff
@@ -384,6 +384,40 @@ The IP address that you choose must be a valid IPv4 or IPv6 address from within
 If you try to create a Service with an invalid clusterIP address value, the API
 server will return a 422 HTTP status code to indicate that there's a problem.
 
+## Traffic policies
+
+### External traffic policy
+
+You can set the `spec.externalTrafficPolicy` field to control how traffic from external sources is routed.
+Valid values are `Cluster` and `Local`. Set the field to `Cluster` to route external traffic to all ready endpoints
+and `Local` to only route to ready node-local endpoints. If the traffic policy is `Local` and there are are no node-local
+endpoints, the kube-proxy does not forward any traffic for the relevant Service.
+
+{{< note >}}
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+If you enable the `ProxyTerminatingEndpoints`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`ProxyTerminatingEndpoints` for the kube-proxy, the kube-proxy checks if the node
```

Comment:
Does this affect InternalTrafficPolicy?

Reply:
Nit: I'd usually just say kube-proxy without the "the".

Reply:
Our (SIG Docs) style guide prefers the “the”; mostly for [starting sentences](https://kubernetes.io/docs/contribute/style/style-guide/#starting-a-sentence-with-a-component-tool-or-component-name) but also elsewhere.

Reply:
> Does this affect InternalTrafficPolicy?

One for SIG Network to answer, I think.

Reply:
ping @andrewsykim

Reply:
It is mentioned in the release note and I've verified it, it only affects ExternalTrafficPolicy https://github.com/kubernetes/kubernetes/pull/97238